File: Front-end/app/common/service/employee.service.js
export default function employeeService($http) {
    var service = this;

    service.getEmployeesByProjectId = function (projectId) {
        return $http({
            method: "GET",
            url: EM_REST_SERVER_URI + "/employees/?projectId=" + projectId
        }).then(function (response) {
            return response.data;
        });
    };
}

File: Back-end/src/main/java/by/bsuir/em/service/EmployeeService.java
package by.bsuir.em.service;

import by.bsuir.em.dto.EmployeeDto;

import java.util.List;

public interface EmployeeService {
    List<EmployeeDto> getEmployeesByProjectId(Long projectId);
    EmployeeDto getEmployeeById(Long id);
}
